public/app/plugins/panel/barchart/BarChartPanel.tsx

```diff
@@ -55,8 +65,31 @@ interface Props extends PanelProps<PanelOptions> {}
 
 export const BarChartPanel: React.FunctionComponent<Props> = ({ data, options, width, height, timeZone, id }) => {
   const theme = useTheme2();
+  const styles = useStyles2(getStyles);
```

Comment:
something funky here -- either pass in the theme, or remove it from the function signature

Reply:
This is normal for how we include styles in other files (see `getStyles` func at end of this file)

Reply:
:exploding_head: you are right :)